bin/pycbc_live

```diff
@@ -448,8 +492,15 @@ class LiveEventManager(object):
             if self.ifar_upload_threshold < ifar \
                     and not any(nearby_coincs) \
                     and self.run_snr_optimization:
-                self.setup_optimize_snr(single, [ifo], bank, fname,
-                                        data_reader, gid)
+                self.setup_optimize_snr(
+                    single,
+                    [ifo],
+                    [ifo],
```

Comment:
I think this will still do the optimisation only on the single-detector that produced the event - I think we get all of them if we use `active`?

Reply:
The regression that this PR intends to fix is strictly related to coincident candidates, but yup, I noticed this too. I guess I should have asked in #4075, but what is the intention here? Do we want to use all available data for the SNR opt, or just the data with a detectable signal?